content/en/docs/security/config-best-practices.md

```diff
@@ -0,0 +1,234 @@
+to learn how to add the `redaction` processor to your Collector configuration.
+
+### Safeguard resource utilization
+
+After implementing safeguards for resource utilization in your
+[hosting infrastructure](/docs/security/hosting-best-practices/), consider also
+adding these safeguards to your OpenTelemetry Collector configuration.
+
+Batching your telemetry and limiting the memory available to your Collector can
+prevent out-of-memory errors and usage spikes. You can also handle traffic
+spikes by adjusting queue sizes to manage memory usage while avoiding data loss.
+For example, use the
+[`exporterhelper`](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md)
+to manage queue size for your `otlp` exporter:
+
+```yaml
+exporters:
+  otlp:
+    endpoint: <ENDPOINT>
+    sending_queue:
+      queue_size: 800
+```
+
+Filtering unwanted telemetry is another way you can protect your Collector's
+resources. Not only does filtering protect your Collector instance, but it also
+reduces the load on your backend. You can use the
+[`filter` processor](/docs/collector/transforming-telemetry/#basic-filtering) to
+drop logs, metrics, and spans you don't need. For example, here's a
+configuration that drops non-HTTP spans:
+
+```yaml
+processors:
+  filter:
+    error_mode: ignore
+    traces:
+      span:
+        - attributes["http.request.method"] == nil
+```
+
+You can also configure your components with appropriate timeout and retry
+limits. These limits should allow your Collector to handle failures without
+accumulating too much data in memory. See the
+[`exporterhelper` documentation](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md)
+for more information.
+
+Finally, consider using compression with your exporters to reduce the send size
+of your data and conserve network and CPU resources. By default, the
+[`otlp` exporter](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/otlpexporter)
+uses `gzip` compression.
+
+## Extensions
+
+While receivers, processors, and exporters handle telemetry directly, extensions
+serve different needs.
+
+<!--- TODO: Extensions SHOULD NOT expose sensitive health or telemetry data. How? What can you do? -->
+
+### Health and telemetry
+
+Extensions are available for health check information, Collector metrics and
+traces, and generating and collecting profiling data. When enabled with their
+default settings, all of these extensions, except the health check extension,
+are accessible only locally to the OpenTelemetry Collector. Take care to protect
+sensitive information when configuring these extensions for remote access, as
+they might expose it accidentally.
+
+### Collector's internal telemetry
+
+<!--- INSERT RECOMMENDATIONS HERE. For example:
+
+1. Remove zPages.
+1. Remove configuration endpoints.
+-->
```

Comment:
I would remove this for now and add it back later (we can create an issue for this). I don't think we necessarily need this for the first version

Reply:
Removed in 961ac47. I also created #5429 so we remember to iterate.